### DISCUSSION TITLE
Which one do I download? 48 or 60?

### DISCUSSION THREAD

**Discussion by DarkZennen** [Rem0o/FanControl.Releases#1197] [Q&A]
Do I download fancontrolnet48 or fancontrolnet60?

**Comment by Rem0o** [comment]
If you don't know the difference, go for 4.8.

**Reply by DarkZennen** [reply]
Do you know what the differences are? It doesn't say anything about what the differences are in the description.

**Reply by Rem0o** [reply]
It's the version of the .NET Framework runtime used. 

One uses 4.8 (legacy, comes with Windows), the other uses 6.0 the latest one, does not come with Windows.

**Reply by Rafee-M** [reply]
> It's the version of the .NET Framework runtime used.
> 
> One uses 4.8 (legacy, comes with Windows), the other uses 6.0 the latest one, does not come with Windows.

Sorry for replying in this old thread, but is there a performance difference or is it just for compatibility?

**Reply by Rem0o** [reply]
In theory the .NET 7 version should be "faster", but I haven't seen much of a difference in practice, both being very low on resources when the app is in the background with the UI closed.

**Reply by Rafee-M** [reply]
Oh okay, thanks a lot for the response!

**Comment by DarkZennen** [comment]
Okay awesome thank you for letting me know! I just went ahead and used 60. I used fan control before the new ui and I'm muchhhh happier with with it! It's such an amazing piece of software
